tests/integration/test_microk8s.py

```diff
@@ -34,6 +36,33 @@ async def test_microk8s_cluster(e: OpsTest, series: str, cp_units: int, worker_u
         )
     )
 
+    await e.model.wait_for_idle([application_name], timeout=60 * 60)
```

Comment:
Why is the RBAC test performed before the worker nodes join the cluster?

Reply:
As RBAC is only defined for control plane nodes, we try to test this only when the cluster only has control plane nodes. But will the position of the tests before or after joining the worker node affect the result?

Reply:
Addressed these.